modules/twig/src/filters/Video.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace modules\twig\filters;
+
+use modules\twig\abstracts\Filter;
+
+use Craft;
+use craft\elements\Asset;
+
+class Video extends Filter
+{
+    public function getFilters()
+    {
+        return [
+            'video' => 'videoTag',
+            'video_url' => 'videoUrl'
+        ];
+    }
+
+    public function videoUrl($asset, $options = [])
+    {
+        if (empty($asset) || !$asset instanceof Asset) {
+            if ($this->isDevMode()) {
+                throw new \Exception('Input value is null or not a craft\elements\Asset instance.');
+            }
+            return '';
+        }
+
+        $config = Craft::$app->config->getConfigFromFile('general');
+        $src = $asset->getUrl();
+
+        return $src;
+    }
+
+    public function videoTag($asset, $options = [], $track = [])
+    {
+        if (empty($asset) || !$asset instanceof Asset) {
+            if ($this->isDevMode()) {
+                throw new \Exception('Input value is null or not a craft\elements\Asset instance.');
+            }
+            return '';
+        }
+
+        $src = $this->videoUrl($asset);
+        $type = $asset->mimeType;
+
+        $computedAttributes = [];
+        $computedTrackAttributes = [];
+
+        foreach ($options as $key => $value) {
+            if ($value !== 0 && $value !== 'false' && $value !== false) {
```

Comment:
use !empty($value) instead of this.

Reply:
Come to think of it, an empty string should probably be allowed, since a lot of video attributes don't require any values.

Reply:
I think we should not do any form of validation in the attributes, let the dev deal with in on it's own since he is provinding them so I think we should remote the condition here and just loop and assign attributes.

Reply:
Fair enough.

Reply:
Nevermind my comment above, I think the way it is now is fine.

It allows to programmatically add or remove an attribute by attributing it `true` or `false`, which wouldn't work if we removed the condition.